xcube/core/geom.py

```diff
@@ -183,6 +184,9 @@ def mask_dataset_by_geometry(dataset: xr.Dataset,
         (but still may be clipped).
     :param no_clip: If True, the function will not clip the dataset before masking, this is, the
         returned dataset will have the same dimension size as the given *dataset*.
+    :param all_touched: If True, all pixels touched by geometries will be
+        burned in. If False, only pixels whose center is within the polygon
```

Comment:
Please don't use unclear "burned in" metaphor.

Reply:
``